### PR TITLE
Added buildscript for archiver package

### PR DESCRIPTION
**Pull request by saurabhghumnar**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ? 

Package Name: archiver
Package Version: v3.5.0
Package URL: [archiver](https://github.com/mholt/archiver )

Tested on: UBI 8.4
Build Errors: None
Test Errors: None

### CODE REVIEW

File: a/archiver/archiver_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  rm -rf archiver
+fi
+
+# Dependency installation
+dnf install -y go git
+
+mkdir -p /home/tester/go
+export GOPATH=/home/tester/go
+
+cd $GOPATH
```
Comment (Siddhesh-Ghadi) [inline a/archiver/archiver_ubi_8.4.sh:39]: Can we clone the package to `$GOPATH/src/github.com/mholt/` ?

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]

**Review by Siddhesh-Ghadi — APPROVED** [review]
LGTM. Addressed changes requested in  #1118.

**Comment by Siddhesh-Ghadi** [comment]
Please squash the commits. @saurabhghumnar

**Comment by saurabhghumnar** [comment]
> Please squash the commits. @saurabhghumnar

Squashed previous commits. Can be merged now

**Pull request merged**